sql/handler.h

```diff
@@ -3835,6 +3838,12 @@ class handler :public Sql_alloc
    notify_table_changed();
  }
 
+/* return false, mean it can't instant alter default */
+virtual bool check_instant_alter(const Alter_inplace_info* inplace_info)
+{
+	return false;
+}
+
```

Comment:
Use Doxygen markup in the comment: `/**` and `@return`.
Omit the parameter name of the unused parameter, to avoid any compiler warning:
```
virtual bool check_instant_alter(const Alter_inplace_info*) const
```
Yes, I would declare the method `const`, unless there is a very strong reason for it to modify `this`. Based on the method name, there should be no such reason.

Can you write a function comment that properly explains the purpose of this virtual method? Maybe you could use `@retval false` and `@retval true` in that comment, explaining what each return value would mean.

Do we really need this new method? Can the same check be performed in `check_if_supported_inplace_alter()`?